covid_graphs/covid_graphs/formula_test.py

```diff
@@ -4,11 +4,12 @@
 import numpy as np
 import pytest
 
+from .country_graph import _create_trace
```

Comment:
It is weird to have this test to depend on `country_graph`, but this should be solved if the trace creation functionality is moved.

Reply:
Done